includes/WikiPage.php

```diff
@@ -1297,6 +1297,8 @@ public function doEdit( $text, $summary, $flags = 0, $baseRevId = false, $user =
 
 		$flags = $this->checkFlags( $flags );
 
+		Hooks::run( 'ArticleFilesRefresh', [ $this->mTitle ] );
```

Comment:
Can we just use the `ArticleSave` hook instead of adding a new hook? The title can be retrieved from the `ArticleSave` parameters so it should work the same.